diff --git a/dist/cjs/api/types/TurnCreatedEvent.d.ts b/dist/cjs/api/types/TurnCreatedEvent.d.ts
index 3bae403c35a608f510f6693ba13f4947c9470b27..3ef48c579e8c962fa09f83d008ce798167d898a4 100644
--- a/dist/cjs/api/types/TurnCreatedEvent.d.ts
+++ b/dist/cjs/api/types/TurnCreatedEvent.d.ts
@@ -7,7 +7,7 @@ export interface TurnCreatedEvent {
     previousTurnId?: string;
     input?: TrueFoundryGateway.TurnInputItem[];
     state: TrueFoundryGateway.TurnStateRunning;
-    createdBy: TrueFoundryGateway.Subject;
+    createdBy?: TrueFoundryGateway.Subject;
     createdAt: string;
     threadId?: string;
 }
diff --git a/dist/cjs/serialization/types/TurnCreatedEvent.d.ts b/dist/cjs/serialization/types/TurnCreatedEvent.d.ts
index 9f4867135c22f3ee9b7851a4ec1ef766fb7bc301..5ee91460dfeb8385fe485019554f5f13f2f859fd 100644
--- a/dist/cjs/serialization/types/TurnCreatedEvent.d.ts
+++ b/dist/cjs/serialization/types/TurnCreatedEvent.d.ts
@@ -13,7 +13,7 @@ export declare namespace TurnCreatedEvent {
         previous_turn_id?: string | null;
         input?: TurnInputItem.Raw[] | null;
         state: TurnStateRunning.Raw;
-        created_by: Subject.Raw;
+        created_by?: Subject.Raw | null;
         created_at: string;
         thread_id?: string | null;
     }
diff --git a/dist/cjs/serialization/types/TurnCreatedEvent.js b/dist/cjs/serialization/types/TurnCreatedEvent.js
index f61ed6f67e6a2db84b1e850d8918b86b96ad9807..8ce0ed45796a4019a1db6026b1674791b05fc92a 100644
--- a/dist/cjs/serialization/types/TurnCreatedEvent.js
+++ b/dist/cjs/serialization/types/TurnCreatedEvent.js
@@ -46,7 +46,7 @@ exports.TurnCreatedEvent = core.serialization.object({
     previousTurnId: core.serialization.property("previous_turn_id", core.serialization.string().optional()),
     input: core.serialization.list(TurnInputItem_js_1.TurnInputItem).optional(),
     state: TurnStateRunning_js_1.TurnStateRunning,
-    createdBy: core.serialization.property("created_by", Subject_js_1.Subject),
+    createdBy: core.serialization.property("created_by", Subject_js_1.Subject.optional()),
     createdAt: core.serialization.property("created_at", core.serialization.string()),
     threadId: core.serialization.property("thread_id", core.serialization.string().optional()),
 });
diff --git a/dist/esm/api/types/TurnCreatedEvent.d.mts b/dist/esm/api/types/TurnCreatedEvent.d.mts
index f0bb125d2eeccc2b23b0256a91d395a8a8641b55..7a5f70c8cc614c98f1e3c06d8294cca616f1709f 100644
--- a/dist/esm/api/types/TurnCreatedEvent.d.mts
+++ b/dist/esm/api/types/TurnCreatedEvent.d.mts
@@ -7,7 +7,7 @@ export interface TurnCreatedEvent {
     previousTurnId?: string;
     input?: TrueFoundryGateway.TurnInputItem[];
     state: TrueFoundryGateway.TurnStateRunning;
-    createdBy: TrueFoundryGateway.Subject;
+    createdBy?: TrueFoundryGateway.Subject;
     createdAt: string;
     threadId?: string;
 }
diff --git a/dist/esm/serialization/types/TurnCreatedEvent.d.mts b/dist/esm/serialization/types/TurnCreatedEvent.d.mts
index d55a14dda8ba0c7ab3fdb1c0a4228c7c9c58abc8..99590e0c4833a29742d6c22568846b26761e9693 100644
--- a/dist/esm/serialization/types/TurnCreatedEvent.d.mts
+++ b/dist/esm/serialization/types/TurnCreatedEvent.d.mts
@@ -13,7 +13,7 @@ export declare namespace TurnCreatedEvent {
         previous_turn_id?: string | null;
         input?: TurnInputItem.Raw[] | null;
         state: TurnStateRunning.Raw;
-        created_by: Subject.Raw;
+        created_by?: Subject.Raw | null;
         created_at: string;
         thread_id?: string | null;
     }
diff --git a/dist/esm/serialization/types/TurnCreatedEvent.mjs b/dist/esm/serialization/types/TurnCreatedEvent.mjs
index f83da50aa6b211d84a4f0f79828e908d11582535..cd145ee488e035d83a2067acd73aef4c82c75bde 100644
--- a/dist/esm/serialization/types/TurnCreatedEvent.mjs
+++ b/dist/esm/serialization/types/TurnCreatedEvent.mjs
@@ -10,7 +10,7 @@ export const TurnCreatedEvent = core.serialization.object({
     previousTurnId: core.serialization.property("previous_turn_id", core.serialization.string().optional()),
     input: core.serialization.list(TurnInputItem).optional(),
     state: TurnStateRunning,
-    createdBy: core.serialization.property("created_by", Subject),
+    createdBy: core.serialization.property("created_by", Subject.optional()),
     createdAt: core.serialization.property("created_at", core.serialization.string()),
     threadId: core.serialization.property("thread_id", core.serialization.string().optional()),
 });
